FrontDesk/src/FrontDesk.Api/FrontDesk.Api.csproj

```diff
@@ -7,6 +7,136 @@
     <DockerfileContext>..\..</DockerfileContext>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Images\Clients\Steve_Smith.jpg" />
```

Comment:
I wonder if there's a better way to manage these? Can't they just go in wwwroot?

Reply:
wwwroot in blazor mean static in blazor so api will also be removed for images?

Reply:
This isn't the Blazor project though, it's the API project, right?